Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION=6.3.0
+VERSION=6.4.0
```

Comment:
curious if this is a breaking change?
user will not see some information in the command line interface, right.
could be considered breaking.